src/istio/control/http/attributes_builder.cc

```diff
@@ -274,6 +276,17 @@ void AttributesBuilder::ExtractReportAttributes(ReportData *report_data) {
   }
 
   builder.FlattenMapOfStringToStruct(report_data->GetDynamicFilterState());
+
+  std::string vs;
+  if (report_data->FindVirtualService(&vs)) {
+    // TODO: add name, namespace attributes
```

Comment:
Can that be derived at the recipient side on-demand instead? Would save on bytes transferred.